Api/Src/Models/GradeSheet.ts

```diff
@@ -28,6 +31,11 @@ const GradeSheetSchema = new mongoose.Schema({
         ref: 'Project',
```

Comment:
There should be ref to TeamProject, not Project

Reply:
Thanks, corrected 👍